.bit/responses/5.4-Week 5 Step 4.md

```diff
@@ -0,0 +1,53 @@
+---
+files: n/a
+stepType: IssueComment
+scripts: n/a
+week: 5
+step: 4
+name: Week 5 Step 4
+---
+Week 5 Step 4 ◯◯◯⬤◯◯ | 🕐 Estimated completion: 25 minutes
+
+## Timeline
+
+### ✅  Task:
+
+- [ ] Develop a draft of your timeline in a new file called "timeline.md" in the projects folder
+- [ ] Ask a mentor to review your gameplan
+- [ ] Once your timeline is mentor approved, convert your timeline into issues in this repo
+- [ ] Commit your timeline.md file to the  `week5` branch to move on
+
+### Developing a timeline
+
+Before you start to code, you need to have a overaching plan on how you will build your project. It is completely okay (in fact, probably inevitable), for your plan to change once you start developing, but you still need a place to start.
+
+### Where to start
+
+Look back at your checklist and decide where you want to start developing your project. Do you want to start with:
+- UI of the app/webapp
+- Azure function
+- API implementation
+
+The best approach recommended by our mentors is to "start small then build big." If you're really unsure, ask your mentor!
+
+### Break it down
+
+The worst part of any task is not knowing when it's going to end. When you're developing your own timeline, make sure that each task doesn't take you more than 1-2 hours. The recommendation is to have tasks that can individually be completed in less than 25mins.
+
+In reality, they might end up taking more time (especially with debugging involved) but try your best to estimate.
+
+### Scheduling deadlines
+
+Speak with your mentor to make sure you can develop your timeline to maximize your timeline. Keep in mind that there will be many times when you might not be able to meet your own deadlines. How will you deal with this? Do you have enough buffer time allotted in your timeline for when things don't go according to plan?
+
+Sometimes you might become lucky and end up having extra time on your hands. How do you plan on maximizing the impact of this time so you can ship the best version of your project at the end?
+
+Be clearn on how to manage your own time and hold yourself accountable! Ask your peers and friends for help if you need it.
+
+### Getting feedback
+
+Your mentor knows best! They are professionals who have been working in the tech industry on projects much more complex and that have covered all sorts of timelines and team sizes. Get their feedback on your plan and see if they have any recommendations on how you can build your project more efficiently.
+
+### Converting tasks into issues
```

Comment:
We need examples for each of these (maybe templates?) - that way it seems less overwhelming